modules/worker-utils/src/lib/worker-farm/worker-body.ts

```diff
@@ -2,15 +2,22 @@ import type {WorkerMessageData, WorkerMessageType, WorkerMessagePayload} from '.
 import {getTransferList} from '../worker-utils/get-transfer-list';
 
 /** Vile hack to defeat over-zealous bundlers from stripping out the require */
-function getParentPort() {
+async function getParentPort() {
   // const isNode = globalThis.process;
   let parentPort;
   try {
     // prettier-ignore
     eval('globalThis.parentPort = require(\'worker_threads\').parentPort'); // eslint-disable-line no-eval
     parentPort = globalThis.parentPort;
-    // eslint-disable-next-line no-empty
-  } catch {}
+  } catch {
+    try {
+      // prettier-ignore
+      eval('globalThis.workerThreadsPromise = import(\'worker_threads\')'); // eslint-disable-line no-eval
+      const workerThreads = await globalThis.workerThreadsPromise;
+      parentPort = workerThreads.parentPort;
+      // eslint-disable-next-line no-empty
+    } catch {}
```

Comment:
Should we at least `console.error(error.message)` here? Completely swallowing errors can lead to very exhausting debugging sessions later on.